debian/copyright

```diff
@@ -1,22 +1,381 @@
-This is syslog-ng, written and maintained by  <bazsi@balabit.hu>
-on Sat, 03 Mar 2007 16:07:59 +0100
-
-The original source can always be found at:
-	http://www.balabit.hu/downloads/files/syslog-ng
-
-Copyright (C) 2002-2011 BalaBit IT Security Ltd
-Copyright (C) 1998-2011 Balazs Scheidler
-
- * This program is free software; you can redistribute it and/or modify
- * it under the terms of the GNU General Public License as published by
- * the Free Software Foundation; either version 2 of the License, or
- * (at your option) any later version.
- *
- * This program is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- * GNU General Public License for more details.
- *
- * You should have received a copy of the GNU General Public License
- * along with this program; if not, write to the Free Software
- * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301 USA
+Format: http://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: syslog-ng
+Upstream-Contact: Syslog-ng users' and developers' mailing list <syslog-ng@lists.balabit.hu>
+Source: git://github.com/balabit/syslog-ng.git
+Copyright: Copyright (C) Balázs Scheidler
+           Copyright (C) BalaBit IT Security Ltd.
+
```

Comment:
I've added copyright holder extraction & gathering as an experimental
feature to the checker from the start. It outputs it at the end if given
enough verbosity level, and I recall it also produces a list file with it.
A little tweaking of the output (probably a single line of sed) would
produce the desired effects.

On Sun, Jul 17, 2016 at 11:29 AM, Balazs Scheidler <notifications@github.com
> wrote:

> In debian/copyright
> <https://github.com/balabit/syslog-ng/pull/1125#discussion_r71079409>:
>
> > - *
> > - * This program is distributed in the hope that it will be useful,
> > - * but WITHOUT ANY WARRANTY; without even the implied warranty of
> > - * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
> > - * GNU General Public License for more details.
> > - *
> > - * You should have received a copy of the GNU General Public License
> > - * along with this program; if not, write to the Free Software
> > - * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301 USA
> > +Format: http://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
> > +Upstream-Name: syslog-ng
> > +Upstream-Contact: Syslog-ng users' and developers' mailing list <syslog-ng@lists.balabit.hu>
> > +Source: git://github.com/balabit/syslog-ng.git
> > +Copyright: Copyright (C) Balázs Scheidler
> > +           Copyright (C) BalaBit IT Security Ltd.
> > +
>
> I realised that this lists all copyrights, any chance of using
> tests/copyright/policy somehow? Do we need this specific format?
>
> I wouldn't stop integrating this patch because of this, but maybe drop
> this file until this one is covered.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/balabit/syslog-ng/pull/1125/files/70a98dc2a8f4b7d3f829f937293ad7bf2b367d28#r71079409>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AOobpMgGZMyiwXZeXVAdcIN48BPum4Jxks5qWfYSgaJpZM4JNcax>
> .
>